noir-projects/aztec-nr/aztec/src/macros/functions/self_call_registry.nr

```diff
@@ -1,3 +1,6 @@
+// TODO(F-131): Drop all this and instead just iterate over the functions directly in the `#[aztec]` macro. This
+// approach was already implemented for `#[internal(...)]` functions.
```

Comment:
Unrelated but realized that the registry approach used here is unnecessary and convoluted - we can just directly iterate over the functions and generate the calls accordingly. I already used this approach for generating the calls to internal. Will tackle this in a followup PR.